Libplanet/Address.cs

```diff
@@ -145,15 +146,27 @@ public ImmutableArray<byte> ByteArray
         [Pure]
         public string ToHex()
         {
-            return ByteUtil.Hex(ToByteArray());
+            string hex = ByteUtil.Hex(ToByteArray());
+            byte[] bytes = Encoding.ASCII.GetBytes(hex);
+            byte[] hash = CalculateHash(bytes);
+            string hashHex = ByteUtil.Hex(hash);
+            string address = string.Empty;
+
+            for (var i = 0; i < hex.Length; i++)
```

Comment:
As `string` implements `IEnumerable<char>`, this can be adjusted to use `foreach` instead.

Reply:
It seems to use `i` as the index of `hexHash`.

Reply:
@longfin Fair enough.  This should be as is.  @earlbread Nevermind!